docs/releasing/header.txt

```diff
@@ -1,19 +1,13 @@
-# Copyright 2022 Google LLC
+# Copyright 2023 Google LLC
 #
 # Licensed under the Apache License, Version 2.0 (the "License");
 # you may not use this file except in compliance with the License.
 # You may obtain a copy of the License at
 #
-#      http://www.apache.org/licenses/LICENSE-2.0
+#     https://www.apache.org/licenses/LICENSE-2.0
 #
 # Unless required by applicable law or agreed to in writing, software
 # distributed under the License is distributed on an "AS IS" BASIS,
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-apiVersion: v1
-kind: Namespace
-metadata:
-  name: bank-of-anthos-development
-  labels:
-    istio-injection: enabled # enable ASM sidecar injection on namespace
```

Comment:
**Thought:**
Regarding folder: `/iac/acm-multienv-cicd-anthos-autopilot/overlays`
I see that we're removing `development/namespace.yaml`.
Note that we're also removing the `istio-injection: enabled` label.
So this means that the development manifests will use the `default` namespace. That's fine.
But the development will behave differently (from production and staging) when using Istio.
Namely, users will have to manually label the namespace with `istio-injection: enabled` to have Istio Sidecars auto-injected.

Reply:
Not a big issue. Feel free to ignore.

Reply:
"So this means that the development manifests will use the default namespace"

Yes. It already does, this `bank-of-anthos-development` namespace actually just sat empty. I think it's residual from an earlier thought that we'd use a specific namespace for development work, but that wasn't compatible with the idea of multiple deployments in the same cluster (e.g. PR clusters) so that idea was scrapped.

"Namely, users will have to manually label the default namespace with istio-injection: enabled to have Istio"

That's fine, neither raw istio or ASM are deployed in the development profile.